### PR TITLE
feat: added batch 3 of vapor thermal conductivity

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

**Review by maneetgoyal — APPROVED** [review]

**Comment by maneetgoyal** [comment]
@qur786 Since the process is automated, make a single PR for rest of the compounds.

File: src/data/vapor-thermal-conductivity-constants.ts [hunk 1]
```diff
@@ -713,6 +713,767 @@ export const DIPPRVaporThermalConductivityDictionary: Record<string, DIPPRVaporT
...
+    minimumTemperature: 392.7,
+    maximumTemperature: 1000,
+    thermalConductivityAtMinimumTemperature: 0.01884,
+    thermalConductivityAtMaximumTemperature: 0.095
+  },
+
+  "cis1,2Dimethylcyclohexane": {
+    name: "cis-1,2-Dimethylcyclohexane",
+    formula: "C8H16",
+    CAS: "04-01-2207",
```
Comment (saikat-samanta) [inline src/data/vapor-thermal-conductivity-constants.ts:880]: @qur786 Check this

**Comment by qur786** [comment]
> @qur786 Since the process is automated, make a single PR for rest of the compounds.

Ok sir

File: src/data/vapor-thermal-conductivity-constants.ts [hunk 2]
```diff
@@ -713,6 +713,767 @@ export const DIPPRVaporThermalConductivityDictionary: Record<string, DIPPRVaporT
...
+    C2: 0.9023,
+    C3: 2170000000,
+    minimumTemperature: 327.46,
+    maximumTemperature: 1000,
+    thermalConductivityAtMinimumTemperature: 0.01426,
+    thermalConductivityAtMaximumTemperature: 0.11921
+  },
+
+  "2EthylHexanoicAcid": {
+    name: "2-EthylHexanoicAcid",
```
Comment (saikat-samanta) [inline src/data/vapor-thermal-conductivity-constants.ts:1340]: 2-Ethyl hexanoic acid

**Comment by saikat-samanta** [comment]
@qur786 Resolve conflict

**Review by saikat-samanta — APPROVED** [review]

**Pull request merged**